app/migrations/Version20171227161442.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+class Version20171227161442 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        $this->addSql('CREATE TABLE elections (id INT AUTO_INCREMENT NOT NULL, name VARCHAR(255) NOT NULL, place VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_1BD26F335E237E06 (name), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
+        $this->addSql('CREATE TABLE election_rounds (id INT AUTO_INCREMENT NOT NULL, election_id INT DEFAULT NULL, label VARCHAR(255) NOT NULL, description VARCHAR(255) DEFAULT NULL, date DATE NOT NULL, INDEX IDX_37C02EA0A708DAFF (election_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB');
```

Comment:
election_id is not nullable no ?

Reply:
`nullable` is not an option for ManyToOne AFAIK. Do you know a work around?

Reply:
Ok I didn't know this was settable on the joinColumn config.

Reply:
Fixed.